sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -845,13 +842,11 @@ public void testBuildWrite() {
   }
 
   @Test
-  @Ignore("[BEAM-436] DirectRunner tempLocation configuration insufficient")
   public void testBatchWritePrimitiveDisplayData() throws IOException, InterruptedException {
```

Comment:
Looks like we no longer have the display data this (and `testStreamingWritePrimitiveDisplayData`) test expects because it was disabled for a while. @jkff @dhalperi do you guys know if this is obsolete or needs fixing?